recipe/meta.yaml

```
@@ -29,32 +29,27 @@ outputs:
         - pip
         - python
         - setuptools
-        - sphinx 5.0.2
+        - sphinx >=1.6,<7
         - wheel
       run:
         - python
-        - sphinx >=1.6,<6
-        # Bump docutils to <0.19 instead of <0.18, see https://github.com/readthedocs/sphinx_rtd_theme/blob/master/docs/changelog.rst#dependency-changes-1
+        - sphinx >=1.6,<7
         - docutils <0.19
+        - sphinxcontrib-jquery >=2.0.0,!=3.0.0  # [py>=3]
     test:
       imports:
         - sphinx_rtd_theme
-      # Disable pip check because of sphinx-rtd-theme 1.1.1 has requirement docutils<0.18, but you have docutils 0.18.1.
-      #requires:
-      #  - pip
-      #commands:
-      #  - pip check
+      requires:
+        - pip
+      commands:
+        - pip check
 
   - name: sphinx-rtd-theme
     build:
       noarch: python
     requirements:
       host:
-        - pip
         - python
-        - setuptools
-        - sphinx 5.0.2
-        - wheel
       run:
         - python
         - {{ pin_subpackage(name, exact=True) }}
```

```suggestion
        - {{ pin_subpackage(name, min_pin='x.x.x', max_pin='x.x.x') }}
```
Otherwise it pins up to the hash, and fix a specific python version.